src/content/staking/dvt/index.md

```diff
@@ -0,0 +1,84 @@
+
+### No single points of failure {#no-single-point-of-failure}
+
+When a validator is divided across multiple operators and multiple machines, it can withstand individual hardware and software failures without going offline. The risk of failures can also be reduced by using diverse hardware and software configurations across the nodes in a cluster. This resilience is not available to today's validators - it comes from the DVT layer.
+
+If one of the components of a machine in a cluster goes down (For example, if there are four operators in a validator cluster and one uses a specific client that has a bug), the others ensure that the validator keeps running. 
+
+### Decentralization {#decentralization}
+
+The ideal scenario for Ethereum is to have as many independently operated validators as possible. However, a few staking providers have become very popular and account for a substantial portion of the total staked ETH on the network. DVT can allow these operators to exist while preserving decentralization of stake. This is because the keys for each validator are distributed across many machines and it would take much greater collusion for a validator to turn malicious.
+
+Without DVT, it's easier for staking providers to support only one or two client configurations for all their validators, increasing the impact of a client bug. DVT can be used to spread the risk across multiple client configurations and different hardware, creating resilience through diversity.
+
+Overall, DVT offers the following benefits to Ethereum:
+- Decentralization of Ethereum's proof-of-stake consensus
+- Ensures the liveness of the network
+- Creates validator fault tolerance
+- Trust minimized validator operation
+- Minimized slashing and downtime risks
+- Improves diversity (client, data center, location, regulation, etc.)
+- Enhanced security of validator key management
+
+## DVT use cases {#dvt-use-cases}
+
+DVT has significant implications for the broader staking industry:
+
+### Solo stakers {#solo-stakers}
+
+DVT also enables non-custodial staking by allowing you to distribute your validator key across remote nodes while keeping the full key completely offline. This means home stakers do not necessarily need to outlay for hardware or add to their home electricity bills, while distributing the key shares can help strengthen them against potential hacks.
+
+### Staking as a service (SaaS) {#saas}
+
+Operators (such as staking pools and institutional stakers) managing many validators can use DVT to reduce their risk. By distributing their infrastructure, they can add redundancy to their operations and diversify their infrastructure. 
```

Comment:
> By distributing their infrastructure, they can add redundancy to their operations and diversify their infrastructure.

Little redundant with wording, ie "By distributing their infrastructure they can… …diversify their infrastructure"
Perhaps:

> By distributing their infrastructure, they can add redundancy to their operations, and diversify their risk tolerance.

Reply:
i think it's more about getting hardware diversity into the validator set for added resiliency (e.g. something bricks validators on a specific processor or something)
```suggestion
Operators (such as staking pools and institutional stakers) managing many validators can use DVT to reduce their risk. By distributing their infrastructure, they can add redundancy to their operations and diversify the types of hardware they use. 
```